network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -343,7 +343,10 @@ class ProductPage(FoundationMetadataPageMixin, Page):
 
     @property
     def current_tally(self):
-        return sum(self.votes.get_votes())
+        if self.votes:
+            return sum(self.votes.get_votes())
+        else:
+            return 0
```

Comment:
This should not be necessary... by the time an instance exists, we should be able to rely on the fact that `self.votes` exists.

Reply:
You're right. But I ran into _something_ (cant recall what atm) where the total votes didnt exist yet. Ill look into this more today.

Reply:
Fixed